R/SelfControlledCohort.R

```diff
@@ -261,15 +261,15 @@ runSelfControlledCohort <- function(connectionDetails,
 
   # Fetch results from server:
   sql <- "SELECT * FROM #results"
-  sql <- SqlRender::translateSql(sql,
+  sql <- SqlRender::translate(sql,
                                  targetDialect = connectionDetails$dbms,
                                  oracleTempSchema = oracleTempSchema)$sql
```

Comment:
You'll need to remove the $sql

Reply:
My sincere apologies, I struggled to get the unit tests running in my R session and thought I'd tested the function call elsewhere in my own code. 

Before running `devtools::test()` I set the postgres parameters to a local copy of postgres with eunomia instance as follows:

    options(dbms="postgresql")
    Sys.setenv(CDM5_POSTGRESQL_SERVER = "localhost/test")
    Sys.setenv(CDM5_POSTGRESQL_PORT = 5432)
    Sys.setenv(CDM5_POSTGRESQL_USER = "postgres")
    Sys.setenv(CDM5_POSTGRESQL_PASSWORD = "my password")
    Sys.setenv(CDM5_POSTGRESQL_CDM_SCHEMA = "eunomia")

But then get the following error:

     test-parameterSweep.R:86: error: SCC
     Cannot find 'Scc.sql' in the 'sql/sql_server' folder of the 'SelfControlledCohort' package.

     Backtrace:

     1. SelfControlledCohort:::testAllParams(...) D:\UsersData\JGilber2\SelfControlledCohort/tests/testthat/test- 
     parameterSweep.R:86:2
     2. SelfControlledCohort::runSelfControlledCohort(...) D:\UsersData\JGilber2\SelfControlledCohort/tests/testthat/test-parameterSweep.R:34:12
     3. SqlRender::loadRenderTranslateSql(...) R/SelfControlledCohort.R:226:2
`

So it doesn't seem to be able to find the sql files in the package inst directory. I've built and installed the package in R studio without error and also tried using devtools to install it. Is there something obviously wrong with my configuration here or should I submit an issue?

Reply:
This is an issue with the devtools test function (which doesn't fully build the package, and therefore anything in the inst/ folder isn't available to the package). It does not affect R CMD Check, which executes the unit tests as part of a more integral evaluation of the package. 

If you want to run the unit tests separately from R CMD check, you'll need to disable devtools when testing.

You cannot use Eunomia as a Postgres server since it is a RSQLite server.

I will send you the credentials for the testing server in an e-mail.